README.md

```diff
@@ -165,12 +165,22 @@ library
   default-language:    Haskell2010
 ```
 
-If you want to be able to import `Extra.*` modules when using `mixins` approach,
-you need to list those modules under `mixins` field as well, like this:
+This way, you can't access any other module of relude. If you want to use e.g.
+`Relude.Extra` (which reexports all `Relude.Extra.*` modules), you can add it
+like this:
 
 ```cabal
   mixins:              base hiding (Prelude)
-                     , relude (Relude as Prelude, Relude.Extra.Enum)
+                     , relude (Relude as Prelude, Relude.Extra)
+```
+
+If you want to be able to import every module of relude, you need to mixin `relude`
```

Comment:
Hmm, using `mixin` as the verb looks fun, however could be a little too complicated for understanding for people not that fluent with the `cabal` terminology. I would prefer to be more explicit on what does "to mixin" means, if you don't mind :slightly_smiling_face: